arxiv/license.py

```diff
@@ -0,0 +1,114 @@
+"""arXiv license definitions."""
+
+
+LICENSE_ICON_BASE_URI = '/icons/licenses'
+LICENSES = {
+    'http://arxiv.org/licenses/nonexclusive-distrib/1.0/': {
+        'uri': 'http://arxiv.org/licenses/nonexclusive-distrib/1.0/',
+        'label': 'arXiv.org perpetual, non-exclusive license to '
+        'distribute this article',
+        'note': '(Minimal rights required by arXiv.org. Select this '
+                'unless you understand the implications of other '
+                'licenses)',
+        'order': 1,
+        'is_current': True,
+        'is_valid': True,
+    },
+    'http://creativecommons.org/licenses/by/4.0/': {
+        'uri': 'http://creativecommons.org/licenses/by/4.0/',
+        'label': 'Creative Commons Attribution license',
+        'order': 5,
+        'is_current': True,
+        'is_valid': True,
+        'icon_uri': f'{LICENSE_ICON_BASE_URI}/by-4.0.png'},
+
+    'http://creativecommons.org/licenses/by-sa/4.0/': {
```

Comment:
Not really a problem, but curious why the URIs are used as keys? Seems like this struct could also be a list of dicts?

Reply:
In that case, you could just use list order as de-facto order -- would make it easier to change later, and then you wouldn't need the sort in ``CURRENT_LICENSE_URIS``?